MapboxVision/src/main/java/com/mapbox/vision/VisionManager.kt

```diff
@@ -161,6 +163,7 @@ object VisionManager : BaseVisionManager {
         this.application = application
         this.mapboxToken = mapboxToken
         PreferencesManager.appContext = application
+        metricsManager = MetricsManager.Impl(application)
```

Comment:
Is it really global or we can initialize it in `create`?

Reply:
it's not global and `MetricManager` doesn't have `destroy` method, but it has `attach`/`detach` methods. So I think we can keep it here

Reply:
But if we initialize it in `init` it will be effectively global and reused forever once Vision is initialized